pkg/requestgen/generator_test.go

```diff
@@ -334,8 +334,8 @@ func Test_expand(t *testing.T) {
 			expErr:    nil,
 		},
 		"if using variables, expect to be substituted": {
-			input:     "foo-$POD_NAME-,,${POD_NAMESPACE},$POD_UID",
-			expOutput: "foo-my-pod-name-,,my-namespace,my-pod-uuid",
+			input:     "foo-$POD_NAME-,,${POD_NAMESPACE},$POD_UID,${SERVICE_ACCOUNT_NAME}",
```

Comment:
Can you also add to the e2e test [here](https://github.com/cert-manager/csi-driver/blob/main/test/e2e/suite/cases/variables.go)? We probably want to create a ServiceAccount with a generated name that the Pod gets assigned.

Reply:
Done!